src/subscription_manager/cli_command/register.py

```diff
@@ -204,8 +203,7 @@ def _do_auto_attach(self, consumer):
             # set during service.register() call
             attach.AttachService(self.cp).attach_auto(service_level=None)
         except connection.RestlibException as rest_lib_err:
-            mapped_message: str = ExceptionMapper().get_message(rest_lib_err)
-            print_error(mapped_message)
+            print_error(rest_lib_err)
```

Comment:
sorry for the late notice of this: this part is not correct, as `print_error()` takes a string as parameter; would it be possible to fix it?

Reply:
Thanks for letting me know, here's a PR to resolve this issue: https://github.com/candlepin/subscription-manager/pull/3162